README.md

```diff
@@ -169,8 +169,48 @@ bazel build --enable_runfiles build_pip_pkg
 TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
 pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
 ```
+##### Apple Silicon Support
+**TFRA** Apple Silicon (M1) version requires:
+
+- macOS 12.0.0+
+- Python 3.9
+- tensorflow-macos 2.5.0
+- bazel 4.1.0+
+
+Before installing **TFRA** from source, you need to install tensorflow-macos from Apple. To install the natively supported version of tensorflow-macos, it's recommended to install the [Conda environment](https://github.com/conda-forge/miniforge). 
+
+After installing conda environment, run the following commands in the terminal.
+
+```sh
+# Create a virtual environment
+conda create -n $Your-Environment-Name Python=3.9
+
+# Activate your environment
+conda activate $Your-Environment-Name
+
+# Install TensorFlow macOS dependencies via miniforge
+conda install -c apple tensorflow-deps==2.5.0
+
+# Install base TensorFlow
+python -m pip install tensorflow-macos==2.5.0
+```
```

Comment:
- This part should be dedicated to directly install, so `pip install tensorflow-recommenders-addons` should be added to end of shell block.
- As I know the binary of tensorflow in conda should be different with pure python Tensorflow installation, so here the `recommended` should be required for users, please confirm that.

Reply:
1. Because the section belongs to "installing from source", I only provided installing from source instruction below. I added an optional command to install TFRA arm64 build from PyPi.
2. Changed the `recommended` to `required`. We may need to restrict the tensorflow-macos source to conda until I tested the tensorflow-macos PyPi distribution.